### ISSUE TITLE
Add a clone subcommand

### ISSUE THREAD

**Issue by funkyfuture** [telota/existance#1]
that command would clone an instance and assign a new name and id to the result. this comes handy for testing upgrading paths.

**Comment by funkyfuture** [comment]
@schnoepf , maybe a `revert` command that restores a snapshot that was created during an upgrade would be sufficient for that use-case?

**Comment by schnoepf** [comment]
yes, would be useful. BTW, I was missing in the upgrade instance my monex/instances.xml file which I used to monitor other instances. And I didn't find it in the backup
